app/features/conference/components/Conference.js

```diff
@@ -24,6 +28,7 @@ export default class Conference extends Component {
         script.onerror = console.error;
         script.src = `https://${config.defaultDomain}/external_api.js`;
 
+        // $FlowFixMe
```

Comment:
why is this one needed?

Reply:
It was taking head as null and saying appendChild is not function of null

Reply:
> Cannot call document.head.appendChild because property appendChild is missing in
> null [1].

Reply:
👍